removed dynamic price on publish, liquidity button and trade tab

EnzoVezzaro opened the pull request:
Fixes #.

Changes proposed in this PR:

- Removed trade tab
- Removed dynamic pricing schema in the publish flow
- Removed 'Add liquidity' option

EnzoVezzaro commented on src/components/Asset/AssetActions/Pool/index.tsx:
```diff
@@ -12,43 +11,33 @@ import { usePool } from '@context/Pool'
                 onClick={() => setShowRemove(true)}
                 disabled={!isAssetNetwork}
               >
                 Remove
```
Sure 👍🏽

kremalicious commented on src/components/Publish/Pricing/index.tsx:
```diff
@@ -67,12 +67,6 @@ export default function PricingFields(): ReactElement {
           content: <Fixed content={content.create.fixed} />
         }
       : undefined,
-    appConfig.allowDynamicPricing === 'true'
```
Can also remove both effects in lines 32-61 from this file

EnzoVezzaro replied:
🚀

EnzoVezzaro commented:
> ![image](https://user-images.githubusercontent.com/26000280/178510059-8170788a-8aaa-4869-b000-bd991cb1a2ea.png) fixed price validation seems busted

ok, yes. this last commit broke it

bogdanfazakas commented:
> UPDATE: for some reason removing [these](https://github.com/oceanprotocol/market/pull/1593#discussion_r919002567) lines causes the pricing to not being validated correctly. I reverted these last changes while I'm trying to solve it

Validation seem to work on my end for fre but actual publish doesn't. Maybe it's related to your comment @EnzoVezzaro 
<img width="1627" alt="Screenshot 2022-07-12 at 17 32 46" src="https://user-images.githubusercontent.com/13447142/178515884-1459cb0d-f79d-440e-a2cb-37d0ac63aeca.png">

EnzoVezzaro commented:
> > UPDATE: for some reason removing [these](https://github.com/oceanprotocol/market/pull/1593#discussion_r919002567) lines causes the pricing to not being validated correctly. I reverted these last changes while I'm trying to solve it
> 
> Validation seem to work on my end for fre but actual publish doesn't. Maybe it's related to your comment @EnzoVezzaro <img alt="Screenshot 2022-07-12 at 17 32 46" width="1627" src="https://user-images.githubusercontent.com/13447142/178515884-1459cb0d-f79d-440e-a2cb-37d0ac63aeca.png">

Yeah exactly. I had some leftovers from the default values. All this logic makes it kind of hard to move around, but now it should be at least be ready to go. I tested both free and fixed pricing. Both successful.

AnaLoznianu commented:
LGTM
